GitCommands/Git/GitModule.cs

```diff
@@ -2327,7 +2327,7 @@ public IReadOnlyList<string> GetRemoteNames()
                 .ToList();
         }
 
-        private static readonly Regex _remoteVerboseLineRegex = new(@"^(?<name>[^	]+)\t(?<url>.+?) \((?<direction>fetch|push)\)$", RegexOptions.Compiled);
+        private static readonly Regex _remoteVerboseLineRegex = new(@"^(?<name>[^	]+)\t(?<url>.+?) \((?<direction>fetch|push)\)(?<option>\s*\[[^\]]*])?$", RegexOptions.Compiled);
```

Comment:
Capturing is affecting performance.
More important, I would look for how that data is used.
```suggestion
       // Ignore trailing options
        private static readonly Regex _remoteVerboseLineRegex = new(@"^(?<name>[^	]+)\t(?<url>.+?) \((?<direction>fetch|push)\)(?:\s*\[[^\]]*])?$", RegexOptions.Compiled | RegexOptions.ExplicitCapture);
```

Reply:
> I would look for how that data is used

I guess git uses them automatically.

Reply:
> I guess git uses the option(s) automatically.

I just meant `(?<option`
ExplicitCapture is not set, OK